autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -281,6 +284,12 @@ def _get_base_search_space(
                 raise ValueError('Invalid key in exclude: %s; should be one '
                                  'of %s' % (key, keys))
 
+        if self.search_space_updates is not None and isinstance(self.search_space_updates,
```

Comment:
Ho can it not be an instance of HyperparameterSearchSpaceUpdates? Asking for instance isn't very fast... probably we should query this in the base task once.

Reply:
okay that makes sense.